installer/app/src/views/azure-create-app-tutorial.js.jsx

```diff
@@ -53,6 +53,18 @@ var AzureCreateAppTutorial = React.createClass({
 				</li>
 
 				<li>
+					<img src={AssetPaths["azure-app-1-1.png"]}  style={{
+						width: 659,
+						height: 372
+					}}/>
+					<label>
+						<p>Click on the applications tab and the "ENDPOINTS" button at the bottom</p>
+						<p>Copy your OAuth 2.0 Token Endpoint into the input below</p>
+						<input name="endpoint" type="text" placeholder="https://login.microsoftonline.com/{your-uid}/oauth2/token?api-version=1.0" />
+					</label>
+				</li>
```

Comment:
This step should go after the application is created as the endpoints button doesn't exist if there aren't any applications.